components/Header.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styled from 'styled-components/native';
+import { useFonts } from '@use-expo/font';
+import { AppLoading } from 'expo';
+
+export const Header = () => {
+  const [fontLoaded] = useFonts({
+    'Inter-Regular': require('../assets/fonts/Inter-Regular.otf')
+  });
+
+  if (!fontLoaded) {
+    return <AppLoading />
+  } else {
+    return <Title>JUST DO IT!</Title>
+  };
+};
+
+const Title = styled.Text`
+  font-family: 'Inter-Regular';  
```

Comment:
Hmm, I still get an error that this is not a system font. Maybe add a backup font?